src/components/Members.jsx

```diff
@@ -8,14 +8,7 @@ function Members() {
     { id: 5, firstName: 'Sarah', lastName: 'Lilly-Bruland', title: 'Admin Emeritus', imgUrl: 'https://avatars.githubusercontent.com/u/17872422?v=4' },
     { id: 6, firstName: 'Justin', lastName: 'Woodward', title: 'Admin Emeritus', imgUrl: 'https://avatars.githubusercontent.com/u/17994108?v=4' },
   ]
-  const members = [
-    { id: 0, name: 'John', imgUrl: '/head.png' },
-    { id: 1, name: 'Ismael', imgUrl: '/head.png' },
-    { id: 2, name: 'Nirvignesh', imgUrl: '/head.png' },
-    { id: 3, name: 'Tijana', imgUrl: '/head.png' },
-    { id: 4, name: 'Jennifer', imgUrl: '/head.png' },
-    { id: 5, name: 'Kellan', imgUrl: '/head.png' },
-  ]
+
   return (
     <div>
       <div className="members">
```

Comment:
Suggest changing this to "admins" (or "administrators" if you want) for readability (i.e. in order to more clearly indicate that code-wise, Admins and Contributors are both equally Members). If you do decide to implement this change, make sure to update all relevant names in the CSS.